python/ray/train/tests/test_gpu.py

```diff
@@ -370,25 +381,19 @@ def train_func():
 
         assert next(model.parameters()).is_cuda
 
-        ray.train.save_checkpoint(model=model)
-        ray.train.report(model=model)
+        session.report({}, checkpoint=TorchCheckpoint.from_model(model))
```

Comment:
this should also work for report right? at least for now

Reply:
Well, we are going to deprecate it in the other PR anyway